mars/oscar/core.pyx

```diff
@@ -13,8 +13,10 @@
 # limitations under the License.
 
 import asyncio
+import cython
```

Comment:
``cimport cython`` ?

Reply:
I can use `cimport` instead of `import`.

Thanks. I will fix it.